src/test/java/com/google/api/codegen/testdata/nodejs_doc_main_library.baseline

```diff
@@ -292,16 +291,18 @@ LibraryServiceApi.prototype.matchBookFromArchivedBookName =
  *     // doThingsWith(response)
  * });
  */
-LibraryServiceApi.prototype.createShelf = function createShelf() {
-  var args = arguejs({
-    shelf: Object,
-    options: [gax.CallOptions],
-    callback: [Function]
-  }, arguments);
+LibraryServiceApi.prototype.createShelf = function createShelf(
+    shelf,
+    options,
+    callback) {
+  if (options instanceof Function && callback === undefined) {
```

Comment:
This looks strange to my eyes, and it produces a lot of repeated code. Is this more JS-idiomatic than just forcing people to pass an empty object as `options` if they want to specify `callback` but not `options`?